src/client/index.ts

```diff
@@ -133,6 +141,20 @@ export interface NotificationOptions {
      */
     onSelect?: ActionDeclaration<never, never>|null;
 
+    /**
+     * An {@link NotificationActionResult|action result} to be passed back to the application inside the
+     * {@link NotificationActionEvent|`notification-action`} event fired when the notification the expires.
+     *
+     * This action will be raised if an `expiry` is specified for the notification, when the notification is removed
+     * due to expiry. This action will never be raised if not `expiry` is specified. Note that this action does not
```

Comment:
>This action will be raised if an `expiry` is specified for the notification, when the notification is due to expiry.

>...due to expiring.

Reply:
✅

Reply:
Paragraph rewritten in next push, plus some changes to `onClose` below

Reply:
✅